packages/babel-generator/src/generators/typescript.ts

```diff
@@ -25,6 +25,16 @@ export function TSTypeParameterInstantiation(
 export { TSTypeParameterInstantiation as TSTypeParameterDeclaration };
 
 export function TSTypeParameter(this: Printer, node: t.TSTypeParameter) {
+  if (node.in) {
+    this.word("in");
+    this.space();
+  }
+
+  if (node.out) {
+    this.word("out");
+    this.space();
+  }
+
```

Comment:
Can you add a generator test case to `packages/babel-generator/test/fixtures/typescript`?

Reply:
I'm investigating ci failures in main branch, but I don't know much about ts and jsx.
`<in T>() => {}</in>` will fail when BABEL_8_BREAKING is enabled.
Can someone tell me if this is a problem with the parser or with the test?